<launch>
  <node name="multi_goal" pkg="robot_slam" type="navigation_multi_goals.py" output="screen">
      <param name="goalListX" value="  0.5,  0.5,   0.23, 1.10,   2.90,  1.96,  2.86,   2.11,  2.84, 2.12, 0.195, 1.08, 0.195,  1.00, 0.25"/>
      <param name="goalListY" value=" -0.2,  -0.2, -0.28, -0.26, -0.26, -0.30, -1.60, -1.55,  -2.96, -2.85, -2.70, -2.76, -1.39, -1.50, -3.12"/>
      <param name="goalListYaw" value="90.0, 180.0,  180.0, 0.0,  0.0,   0.0,   0.0,  0.0,   0.0, 180, 180, 180, 180, 180, -150"/>
  </node>
</launch>
